config.yml

```diff
@@ -0,0 +1,99 @@
+defaults:
+  app:
+    publishMetaData: false
+    iterationInterval: 10000
+    maxInstanceLife: '- 96 hours'
+    id: aws-provisioner-dev
+    workerTypeTableName: DevWorkerTypes
+    secretTableName: DevWorkerSecrets
+    workerStateTableName: DevWorkerState
+    awsInstancePubkey: !env AWS_INSTANCE_PUBKEY
+    awsKeyPrefix: "aws-provisioner-dev-managed:"
+    statsComponent: aws-provisioner-dev
+    allowedRegions: "us-west-2,us-east-1,us-west-1"
```

Comment:
`!env:list ALLOWED_REGIONS`

Then configure with ALLOWED_REGIONS="us-west-2 us-east-1 us-west-1"
If you prefer to hard code it,... then just do that in the `production` section...
`allowedRegions: ['us-west-2', 'us-east-1', 'us-west-1']` or even use YAML-style lists :)